Makefile

```diff
@@ -9,7 +9,7 @@ dummyusers:
 	cd contentcuration/ && python manage.py loaddata contentcuration/fixtures/admin_user_token.json
 
 prodceleryworkers:
-	cd contentcuration && celery -A contentcuration worker -l info
+	cd contentcuration/ && celery -A contentcuration worker -l info --concurrency=200 --pool=eventlet
```

Comment:
200 threads?

Reply:
Up to, yes, but it's important to note these are not Python threads but OS level microthreads that are more efficient. Also, example code typically uses 500 or 1000 so I'm being somewhat conservative here. (But should be more than enough for current and even foreseeable future Studio workloads.)

Reply:
@aronasorman I left more comments and a reference about the approach in the ticket referenced above, BTW.